cluster/gce/gci/configure-helper.sh

```diff
@@ -1705,6 +1705,11 @@ function start-kube-apiserver {
     fi
   fi
 
+  if [[ "${ENABLE_APISERVER_DYNAMIC_AUDIT:-}" == "true" ]]; then
+    params+=" --audit-dynamic-configuration"
```

Comment:
This configuration seems to require `--feature-gates=DynamicAuditing=true` according to https://github.com/kubernetes/kubernetes/blob/2f2945732b5821f9d51c6acdfb0e9fabe81bf3a4/staging/src/k8s.io/apiserver/pkg/server/options/audit.go#L603-L606

Don't we need to set it in ${FEATURE_GATES} in this script?

Reply:
I was thinking the only job running this feature would be https://github.com/kubernetes/test-infra/blob/5df2f2f9ff8a75d5e0128889a314241b14749b2b/config/jobs/kubernetes/sig-gcp/sig-gcp-gce-config.yaml#L243 which has all the feature gates on. Let me know if it makes more sense to flip the flag anyway here

Reply:
Ah, I see. Thanks for your explanation. That makes sense for me.